changelogs/unreleased/3115-imjokey

```diff
@@ -0,0 +1 @@
+Add Tencent Cloud S3-Compatible object store providers Support Docs
```

Comment:
With the `Documentation` GitHub tag applied to this PR, you can remove the changelog file.